Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM rails:4.2.6
+
+RUN echo "deb http://us.archive.ubuntu.com/ubuntu precise main universe" | tee -a /etc/apt/sources.list
+
+RUN mkdir /usr/local/node && curl -L https://nodejs.org/dist/v4.4.3/node-v4.4.3-linux-x64.tar.xz | tar Jx -C /usr/local/node --strip-components=1
```

Comment:
It could make sense to use the Nodesource Apt repo here instead. That way you would get GPG signature checking against the package. And you wouldn't have to do the symlinks below.